src/source/Main.scala

```diff
@@ -161,6 +163,10 @@ object Main {
         .text("The prefix for #include of the main C++ header files from Objective-C++ files.")
       opt[String]("objcpp-include-objc-prefix").valueName("<prefix>").foreach(x => objcppIncludeObjcPrefixOptional = Some(x))
         .text("The prefix for #import of the Objective-C header files from Objective-C++ files (default: the same as --objcpp-include-prefix)")
+      opt[String]("cpp-extended-record-include-prefix").valueName("<prefix>").foreach(cppExtendedRecordIncludePrefix = _)
+        .text("The prefix for #include of the extended record header files from C++ files.")
```

Comment:
Might be worth clarifying in these messages that the distinctinction is about the header type, not the type which is including this.  Specifically I'd extend this like to "extended record C++ header (.hpp) files" and the same with the other arg with "extended record Objective-C header (.h) files"